Orange/widgets/visualize/widgetutils/tree/rules.py

```diff
@@ -88,11 +90,11 @@ class ContinuousRule(Rule):
 
     Examples
     --------
-    x ≤ 30
-    >>> rule = ContinuousRule('age', False, 30, inclusive=True)
+    >>> ContinuousRule('age', False, 30, inclusive=True)
+    age ≤ 30.000
 
-    x > 30
-    >>> rule = ContinuousRule('age', True, 30)
+    >>> ContinuousRule('age', True, 30)
+    age > 30.000
```

Comment:
Please see if the `__repr__` implementation is now all right. I've also changed the docstrings to use `print()` since I find this conveys the meaning of the statement

> Rule ∈ [1.000, 3.000)

much clearer than this

> IntervalRule(attr_name='Rule', left_rule=ContinuousRule(attr_name='Rule', greater=True, value=1, inclusive=True), right_rule=ContinuousRule(attr_name='Rule', greater=False, value=3, inclusive=False))

If this is not ok, I can fix it.